index.html

```diff
@@ -137,12 +138,14 @@ <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
     
     <section id="about">
       <div id="background-about">
+        <img src="images/illustration-about me-1-desktop@2x.png" alt="" id="about-img-1" class="desktop">
```

Comment:
- [ ]Kindly as requested by the previous reviewer consider adding this image as a background image because The images that are added to improve the UI of the application should be added as background images.